extensions/claude-code/src/utils/terminalLauncher.ts

```diff
@@ -0,0 +1,202 @@
+import { getPreferenceValues, showToast, Toast } from "@raycast/api";
+import { showFailureToast } from "@raycast/utils";
+import { exec } from "child_process";
+import { promisify } from "util";
+
+const execAsync = promisify(exec);
+
+let isExecuting = false;
+
+interface Preferences {
+  terminalApp: TerminalApp;
+}
+
```

Comment:
```suggestion
```
You don't need to manually set preference types as this is autogenerated in `raycast-env.d.ts`